tests/Repositories/UserRepositoryTest.php

```diff
@@ -82,4 +82,21 @@ public function test_can_de_activate_user()
         $this->assertEquals($user->id, $getUser->id);
         $this->assertFalse($getUser->is_active);
     }
+
+    /** @test */
+    public function test_can_set_user_password()
+    {
+        /** @var User $farhan */
+        $farhan = factory(User::class)->create();
+
+        $response = $this->userRepo->setUserPassword([
+            'password' => 12345678,
+            'user_id'  => $farhan->id,
+        ]);
+
+        $this->assertTrue($response);
+
+        $user = User::findOrFail($farhan->id);
+        $this->assertEquals(1, $user->set_password);
```

Comment:
this should be `assertTrue` and also compare password like `$this->assertTrue(Hash::check($password, $user->password)`